pom.xml

```diff
@@ -153,7 +153,24 @@
             <version>1.16.3</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>io.github.resilience4j</groupId>
+            <artifactId>resilience4j-spring-boot2</artifactId>
+            <version>1.7.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.resilience4j</groupId>
+            <artifactId>resilience4j-reactor</artifactId>
+            <version>1.7.1</version>
+        </dependency>
+        <dependency>
```

Comment:
duplicate :-) @MarcoAGreco

Reply:
Thank you :) solved with [1bcd170](https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/79/commits/1bcd170fedf1812711403b37aaea80c8be58f3d1)